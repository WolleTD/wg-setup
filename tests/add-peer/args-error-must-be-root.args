peer2 aWoSqEDNDrEQ6DIlEgQ2BKKl0fFGA7v/3Qbwu9pbYxg= 172.16.1.2/32
