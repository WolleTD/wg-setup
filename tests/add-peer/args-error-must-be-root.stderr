Error: Must be root to add new peers!
